decidim-proposals/app/helpers/decidim/proposals/admin/proposals_helper.rb

```diff
@@ -115,6 +115,15 @@ def proposals_admin_filter_applied_filters
           end
           html.join(" ").html_safe
         end
+
+        def proposal_tr_tooltip(proposal)
+          i18n_key = if allowed_to?(:create, :proposal_answer) && !proposal.emendation?
```

Comment:
That's a good idea! Although tests are failing now because i18n-tools can't ensure the i18n key is used. Maybe this code could be simplified to this?

```ruby
if allowed_to?(:create, :proposal_answer) && !proposal.emendation?
  t(:answer_proposal, scope: "decidim.proposals.actions")
else
  t(:show, scope: "decidim.proposals.actions")
end
```

Thus we help i18n-tools understand what keys are used too 😄

Reply:
Yes, maybe too tricky for i18n-tools, Thanks 😁